### ISSUE TITLE
Feature Request: Sticky Toolbar for Save Buttons

### ISSUE THREAD

**Issue by hacki65** [gantry/gantry5#1700]
I work a long time since the first version of G5, but it is horrible to save a layout, styles, or particle defaults. If the layout is not to long or not many particle are present it is OK. But this is not often the case.

For every little changes it is necessary scroll up or down to push the save button. PLEASE: this must be changed. My clients said to me, if this will not be improved with a sticky toolbar, they don't want us G5 in the future. And i can not disagree with this wish.

I know this feature request was already posted here but i think this request is very important. So please forgive me :)

**Comment by JoomFX** [comment]
I agree with you completely, that it is a bit annoying to scroll up or down just to see the "Save" button but I just cannot help myself and not say that the following is kind of ridiculous:

> if this will not be improved with a sticky toolbar, they don't want us G5 in the future

Once again, I completely agree with you, and I know that this will be implemented in the future, but what kind of a user should you be to abandon the best framework for such a minor UX "issue"?

**Comment by hacki65** [comment]
The question is when it will be implemented. I told my clients that it will be implemented, but it is not easy to explain why it is not yet implemented.

The sidebar e.g. in the layout view has this sticky function. Why it is so hard to make this also for the toolbar? I know about the problem because of the platforms. But i think it is important to search for a solution.

**Comment by hacki65** [comment]
BTW: i will use Gantry 5 furthermore in the future, because i love it. But i think not it is only a "minor UX issue"!
